src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+.body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
```

Comment:
you could reset those in index.scss file after [import ](https://sass-lang.com/documentation/at-rules/import/)card and stars